src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -271,7 +271,7 @@ protected virtual JToken GetUnclaimedGas(JArray _params)
                 script_hash = null;
             }
             if (script_hash == null)
-                throw new RpcException(-100, "Invalid address");
+                throw new RpcException(RpcError.InvalidParams);
```

Comment:
L240 in this file: we need to check if the specified iterator exists and if not then throw `UnknownIterator` RPC error.